config/config.go

```diff
@@ -21,6 +21,10 @@ var Version = "dev"
 // StripePublishableKey facilitates secure transmission of payment values
 var StripePublishableKey = "pk_live_A6qSWh1v4SrNnrWSftgDcKFQ"
 
+// GitHubClientID facilitates logins with GitHub
+var GitHubClientID = ""
```

Comment:
Should this have the production value hard coded like we do with stripe?

Reply:
That could make sense -- I left it empty because CI will fill it in, but thinking of a user that might pull the code down, that makes sense.

I'm using values from my own GitHub account now, but hard coding it makes sense.